assets/versions/versions.json

```diff
@@ -0,0 +1,4 @@
+{
+  "cliVersion": "v0.5.0",
+  "configFileSchemaVersion": "v1.0.0"
```

Comment:
I'm looking at docker compose, should we keep this simple and use a single digit number?

Reply:
We can go this way, I have no personal preferences on this one.